docs/src/pages/guides/deploy.md

```diff
@@ -335,6 +335,18 @@ Use the following build settings:
 
 Then click the **Save and Deploy** button.
 
+## Render
+
+You can deploy your Astro project on [render](https://render.com/) following these steps:
+
+1. Create a [render.com account](https://dashboard.render.com/) and sign in
+2. Click the "**new+**"-button at the dashboard and select "**Static Site**"
```

Comment:
```suggestion
2. Click the **New +** button from your dashboard and select **Static Site**
```
To match the rest of the copy on this page, we can use bold text to reference the button label; we don’t need bold _and_ quotes